setup.py

```diff
@@ -32,7 +32,11 @@
     start_pos = long_description.find('## Introduction')
     if start_pos >= 0:
         long_description = long_description[start_pos:]
-
+    try:
```

Comment:
I never encounter any text encoding with this script, Did build your wheel under Linux?

Reply:
No, with windows.

Reply:
In https://packaging.python.org/guides/making-a-pypi-friendly-readme/, it mentioned: 
setuptools >= 38.6.0
wheel >= 0.31.0
twine >= 1.11.0

Did your environment satisfy this requirements?

Reply:
I did not use twine, maybe that's why. I used python setup.py bdist_wheel upload.